<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Yibei Jiang - Personal Website</title>
    <link rel="stylesheet" href="styles.css">
</head>
<link rel="alternate icon" class="js-site-favicon" type="image/png" href="https://github.githubassets.com/favicons/favicon.png">
<body>
    <header>
        <h1>Yibei Jiang</h1>
        <nav>
            <ul>
                <li><a href="/publications">Publications</a></li>
                <li><a href="/cv">CV</a></li>
                <li><a href="/projects">Side Projects</a></li>
            </ul>
        </nav>
    </header>

    <div class="container">
        <aside>
            <img src="cafeSelfie.JPG" alt="Yibei Jiang" class="sidebar-image">
        </aside>

        <main>
            <section id="research-interests">
                <h2>Research Interest</h2>
                <p>Machine Learning in Structural Prediction; Molecular Dynamics (MD) Simulations; Drug Design; Computational Structural Biology; TF-DNA Interactions</p>
            </section>

            <section id="biography">
                <h2>Biography</h2>
                <p>
                    I graduated from UCSD with a degree in bioinformatics, where I discovered the power of computational and statistical methods in genomic analysis. This passion led me to Illumina, a leading sequencing company, where I applied my skills as a software engineer and learned about sequencing pipelines. Motivated by this experience, I pursued a Ph.D. in Computational Biology at USC to deepen my expertise.
                    <br><p></p>In my Ph.D. research, I aimed to connect DNA sequence analysis and structural biology, focusing on how high-throughput DNA analysis and 3D structural insights into protein interactions are essential for developing therapeutics. I honed my skills in Molecular Dynamics (MD) simulations, contributing to two published papers on protein-DNA interactions.
                    <br><p></p>Recently, advancements in protein structure prediction using machine learning inspired me to incorporate Deep Learning and Generative Models into my work. Collaborating with my colleagues, I used these tools to analyze protein-DNA and protein-ligand complexes, leading to a publication in Nature Methods. Iâ€™ve now shifted my focus to MD-aided de novo drug design for therapeutic targets, leveraging MD simulations to optimize innovative drug candidates. This is still a new field and the thrill of exploring the unknown in science has been exhilarating and immensely rewarding.
                    <br><p></p>I recently co-produced and analyzed an X-ray crystal structure of a protein-DNA complex, revealing essential molecular mechanisms and structure-function relationships. My MD simulations identified key residues influencing binding, and my hypotheses were validated through experiments by collaborators. I find this process of scientific inquiry enlightening and gratifying, and I look forward to our upcoming publication.
                </p>
            </section>

            <section id="education">
                <h2>Education</h2>
                <p> Ph.D., Quantitative and Computational Biology, University of Southern California, USA, 2019-Present
                    Bachelor of Science, Biology with specialization in Bioinformatics, University of California, San Diego, USA, 2014-2018
                </p>
            </section>
        </main>
    </div>
</body>
</html>
